docs/topics/native/native-c-interop.md

```diff
@@ -1,5 +1,11 @@
 [//]: # (title: Interoperability with C)
 
+> The C libraries import is [Experimental](components-stability.md#stability-levels-explained).
```

Comment:
Objective-C libraries import is also experimental. So maybe we should also include this header in https://kotlinlang.org/docs/native-objc-interop.html? And mention Objective-C explicitly.